src/client/North/Form/NorthForm.jsx

```diff
@@ -58,16 +58,19 @@ const NorthForm = ({ application, applicationIndex, onChange }) => {
             )}
           />
           {pencil && (
-            <Button
-              outline
-              onClick={() => {
-                setRenamingConnector(`north-${application.id}`)
-                setPencil(false)
-              }}
-              className="util-button"
-            >
-              <FaPencilAlt className="oi-icon ms-2" />
-            </Button>
+          <Button
```

Comment:
These changes should also be reflected in the SouthForm.jsx file

Reply:
Yes, I forgot to add the changes to the other instances of the same edit. Will do